lib/index.js

```diff
@@ -52,6 +57,10 @@ Object.defineProperties(Video, {
     enumerable: true,
     value: LocalVideoTrack
   },
+  Logger: {
```

Comment:
This is interesting. Just curious - why do we expose Logger here instead of asking developers `require('loglevel')` themselves? What are the tradeoffs?

Reply:
If user require the module themselves, they won't be able to see the effects of `setLevel` until a page reloads happen. Loglevel uses local storage to save the state and the SDK will pick up the state after page reload. The other downside is that adding a plugin to the loglevel module only work if npm bundles the same loglevel version. So if the user happens to use a different loglevel, they won't be able to intercept the logs.

Reply:
>by state - you mean logLevel previously set right? if they requireed and set logLevel before calling twilio-video APIs then issue #1 can be addressed right?

Yes. If user only uses loglevel to set the level, they can do so freely by just requiring loglevel module themselves.

>regarding loglevel version - How would this work if they are using multiple libraries that depend on loglevel. Say voice and video - if video requires one version and voice another ?

It will still work if the user is only using the `setLevel` APIs because loglevel saves this state to localStorage. However, it won't work if the user wants to intercept the logs. This seems to be a limitation of the loglevel module because it saves its list of logger objects inside the module's scope, not globally.